front/internals/webpack/webpack.config.js

```diff
@@ -29,7 +29,7 @@ dotenv.config({
 const argv = require('yargs').argv;
 const appLocalesMomentPairs = require(path.join(
   process.cwd(),
-  'app/containers/App/constants'
+  'app/containers/App/constants2'
```

Comment:
Why rename this?

Reply:
Because vite was getting confused by this. When you try to import `app/containers/App/constants`, it would go for the `.js` version instead of the `.ts` version, and the `.js` version uses the old export syntax which vite doesn't support

Reply:
Do we know where the `js` import is being used? 🤔 Could we check and remove it if the answer is nowhere?

Reply:
Alternatively, perhaps we can rename the `js` file and leave the other one (`ts`) as constants as I think at the very least it's used more and it would be best to not change the name of it

Reply:
From what I understand, it's only used in the `front/internals/scripts/extract-intl.js` script, where we don't support `.ts` because we're not transpiling anything. 

> Alternatively, perhaps we can rename the js file and leave the other one (ts) as constants

That's what I did :) or do you mean, change it to something else besides `constants2`, like `commonjs-constants`?

Reply:
Hm, I thought I saw `constants2` imported in a couple of other places where I would expect us to use the `ts` import but I might be wrong - a bit hard to scroll though..

I think `commonjs-constants` is clearer name, indeed, so maybe let's go with that?

Reply:
I guess it's also used in `webpack.config.js`, and then in `eslintrc.js` in order to make the linter ignore it. I'll change the name :+1: